WORKSPACE

```diff
@@ -27,10 +27,11 @@ protobuf_deps()
 http_archive(
     name = "io_bazel_rules_go",
     urls = [
-        "https://storage.googleapis.com/bazel-mirror/github.com/bazelbuild/rules_go/releases/download/v0.21.0/rules_go-v0.21.0.tar.gz",
-        "https://github.com/bazelbuild/rules_go/releases/download/v0.21.0/rules_go-v0.21.0.tar.gz",
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/archive/5474789768ede024a294137d8f249d5cb5b60627.tar.gz",
+        "https://github.com/bazelbuild/rules_go/archive/5474789768ede024a294137d8f249d5cb5b60627.tar.gz",
```

Comment:
Would it work to use v0.21.1? (6 days old)

Reply:
Nope, but we already asked if we can have a new release that includes this fix, see https://github.com/bazelbuild/rules_k8s/pull/517#issuecomment-581451960